### PR TITLE
Implementing Movement in Microgravity

### PR DESCRIPTION
**Pull request by Turtlelord26**
I decided for the sake of complexity to allow for a lot of GM Discretion for things like angles and collision physics. I've also focused almost entirely on movement, restricting the effects of weapons fire to increased difficulty of Anchoring oneself.

I have hijacked the 02 slot for nonstandard environmental rules. This is of course subject to change on request.

Remember that this has 'Supplemental Rulebook Material' written all over it.

### CODE REVIEW

File: 02_01_Advanced_Movement:_Microgravity.txt [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+off nearby or the Anchored object being struck by a ship weapon. In those cases, 
+the character has an optional reflex save based on the force of the event to avoid 
+being Launched – failure causes the character to be Launched directly away from the 
+incoming force as described below.
+
+Small explosives or Blunt Melee Strike: 1m per 2 actions and 40 Reflex per 30 points of damage
+Demolition explosives or Vehicle/Ship Weapon: 8m per 2 actions and 50 Reflex per 30 hull points of damage
+
+	A character may spend an Action to Brace for Impact, which grants Advantage
+on any Reflex saves to avoid being Launched until the character takes another Action.
```
Comment (bleehu) [inline 02_01_Advanced_Movement:_Microgravity.txt:42]: I *Love* the brace for impact rule! Brilliant!

File: 02_01_Advanced_Movement:_Microgravity.txt [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+	In the case that the Anchoring surface is a capital ship, station, or 
+similarly high-inertia object, characters are not Launched by strikes against the 
+surface that are more than 50m away except in truly calamitous circumstances, at 
+the DM’s discretion.
+
+	In the case that a character would be Launched toward an adjacent solid 
+object, such as the surface they were Anchored to, that character may make a DC 120 
+reflex save to catch themselves (Anchoring equipment may provide bonuses), in which 
+case they become Anchored again but gain the prone state. Failure causes the character 
+to uncontrolled Bounce. If the Launching event was not a surprise, the character may
```
Comment (bleehu) [inline 02_01_Advanced_Movement:_Microgravity.txt]: Non blocking: "failure causes the character to bounce uncontrollably" flows a little better I think.

File: 02_01_Advanced_Movement:_Microgravity.txt [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+Rough or Curved Surface: +20
+Damaged Surface: +40
+
+Speed:
+Speed equal or less than the character’s Movement Speed: -20
+Speed greater than twice the character’s Movement Speed: +15 * Speed / Movement Speed
+
+Angle of Impact:
+Less than 30 degrees: -15
+Greater than 60 degrees: +30
```
Comment (bleehu) [inline 02_01_Advanced_Movement:_Microgravity.txt]: Can we make explicit that between 30 and 60 the modifier is 0?

**Review by bleehu — APPROVED** [review]
This is awesome! But would it make more sense to put it in the space combat repo?

**Comment by Turtlelord26** [comment]
I decided to put it here because, even though it's about a situation that is most likely occurring in space, it's describing mechanics at ground combat scale. If you would still prefer it go in Space, I can certainly move it.

File: 02_01_Advanced_Movement:_Microgravity.txt (hunk 3 above)
Comment (bleehu) [reply 02_01_Advanced_Movement:_Microgravity.txt]: new change handled this. Thanks Turtle!

**Comment by bleehu** [comment]
shipit!

**Pull request merged**